packages/gawk.rb

```diff
@@ -3,30 +3,18 @@
 class Gawk < Package
   description 'The gawk utility interprets a special-purpose programming language that makes it possible to handle simple data-reformatting jobs with just a few lines of code.'
   homepage 'https://www.gnu.org/software/gawk/'
-  @_ver = '5.1.1'
-  version "#{@_ver}-1"
+  @_ver = '5.2.0'
+  version @_ver
   license 'GPL-2'
   compatibility 'all'
-  source_url 'https://git.savannah.gnu.org/git/gawk.git'
-  git_hashtag "gawk-#{@_ver}"
-
-  binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gawk/5.1.1-1_armv7l/gawk-5.1.1-1-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gawk/5.1.1-1_armv7l/gawk-5.1.1-1-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gawk/5.1.1-1_i686/gawk-5.1.1-1-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gawk/5.1.1-1_x86_64/gawk-5.1.1-1-chromeos-x86_64.tpxz'
-  })
-  binary_sha256({
-    aarch64: '0963d6d7067653da8039fc4182235d7b8b75bb4adc8e2448e122a85d00cb3d93',
-     armv7l: '0963d6d7067653da8039fc4182235d7b8b75bb4adc8e2448e122a85d00cb3d93',
-       i686: '9c6609595c2fa11e1a5e9ddda7e6a592b4040549a37915ebae929ba12fa41fea',
-     x86_64: '2734c6270be8ee586007e4389c762b956f993d7503e9cb7d863c2b675bde9e2b'
-  })
+  source_url 'https://ftpmirror.gnu.org/gawk/gawk-5.2.0.tar.xz'
+  source_sha256 'e4ddbad1c2ef10e8e815ca80208d0162d4c983e6cca16f925e8418632d639018'
 
   depends_on 'readline' => :build
   depends_on 'ncurses'
   depends_on 'mpfr'
   depends_on 'gmp'
+  depends_on 'libsigsegv' unless ARCH == 'i686'
```

Comment:
Why isn't this used on i686?

Reply:
it apparently causes segfaults. I haven't tested it myself, I just saw the comments and followed suit